polaris-react/src/components/Filters/Filters.scss

```
@@ -1,295 +1,261 @@
-  );
-  // stylelint-enable
-  padding: var(--p-space-2);
+.FiltersInner {
+  overflow: auto;
+  white-space: nowrap;
+  padding: var(--p-space-3) var(--p-space-4) var(--p-space-5);
 }
 
-.FiltersMobileContainerContent {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: calc(100% - #{$list-filters-header-height});
-  padding: var(--p-space-2);
+.hideQueryField .FiltersInner {
+  flex: 1;
 }
 
-.FiltersContainerFooter {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  position: absolute;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  bottom: 0;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  padding: var(--p-space-4) var(--p-space-5);
-  border-top: var(--p-border-divider);
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $list-filters-footer-height;
-  box-sizing: border-box;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: flex;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  align-items: center;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: space-between;
+// stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+@media #{$p-breakpoints-md-up} {
+  .FiltersInner {
+    overflow: visible;
+    flex-wrap: wrap;
+    gap: var(--p-space-2);
+    padding: calc(var(--p-space-2) - var(--p-space-05)) var(--p-space-2);
+  }
 }
 
-.FiltersMobileContainerFooter {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  padding: var(--p-space-4) var(--p-space-4);
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $list-filters-footer-height;
-  box-sizing: border-box;
+.AddFilter {
+  background: var(--p-surface-subdued);
+  border-radius: $border-radius;
+  border: var(--p-border-subdued) dashed $border-width;
+  padding: calc(var(--p-space-1) - var(--p-space-05))
+    calc(var(--p-space-2) - var(--p-space-025))
+    calc(var(--p-space-1) - var(--p-space-05))
+    calc(var(--p-space-3) - var(--p-space-025));
+  cursor: pointer;
+  color: var(--p-text);
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   display: flex;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   align-items: center;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: space-between;
-}
-
-.EmptyFooterState {
-  border-top: var(--p-border-divider);
-  padding-top: var(--p-space-4);
+  justify-content: center;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
+  font-size: 15px;
```

```suggestion
  font-size: var(--p-font-size-100);
```

Could this go down to 14px?